### PR TITLE
dataMigrate install uses prisma migrate command

### PR DESCRIPTION
**Pull request by dthyresson**
The legacy dataMigrate install invoked `yarn rw db save` which is deprecated and not longer works.

This PR changes the command to use:

`yarn rw prisma migrate dev --create-only` and passes the migration name as `--name=create_data_migrations`.

Note that I have tested that `yarn prisma migrate dev --create-only --name=create_data_migrations --preview-feature --schema "...api/db/schema.prisma"` works as expected, so I know can pass the migration name this way.

Unfortunately due to other prisma output issues (? https://github.com/redwoodjs/redwood/issues/1786 ?) when running the command in the current `0.25.1-canary.2` release with this PR applied:

`yarn rw dataMigrate install`

I get the error:

```
✔ Creating dataMigrations directory...
  ✔ Adding RW_DataMigration model to schema.prisma...
  ✖ Create db migration...
    → info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this comm
…
    One more thing...
Command failed with exit code 1: yarn rw prisma migrate dev --create-only --name=create_data_migrations
......
      else throw err
           ^

TypeError: innerYargs._parseArgs is not a function
```

This PR also updates the url to link to the install command on the docs site.

### CODE REVIEW

File: packages/cli/src/commands/dataMigrate/install.js
Comment (dac09) [inline packages/cli/src/commands/dataMigrate/install.js:49]: Hey @dthyresson, adding `stdio: 'inherit'` here will prevent the issue happening incase these commands show up an interactive prompt

Comment (dthyresson) [reply packages/cli/src/commands/dataMigrate/install.js:49]: Unfortunately that didn't help. Still same error.

In fact I get that error on every

`yarn rw prisma *`

now.

`yarn rw prisma studio`
`yarn rw prisma migrate dev`

Maybe I will wait until the other prisma command are merged in and try again.

Comment (dac09) [reply packages/cli/src/commands/dataMigrate/install.js:49]: Yeah the other PR will fix the interactive issue. Just saying adding that here will also prevent future issues (if there's interactivity even with the flags)

**Review by dac09 — APPROVED** [review]
🍔

**Comment by jvanbaarsen** [comment]
Related: https://github.com/redwoodjs/redwood/issues/1794

**Comment by thedavidprice** [comment]
Just catching up here. Do we need @cannikin eye's on this one before merging?

**Comment by dthyresson** [comment]
> Do we need @cannikin eye's on this one before merging?

I think it is good but always good to have @cannikin give it a look. And then we can each test in a new and existing app.

**Review by cannikin — APPROVED** [review]

**Comment by cannikin** [comment]
Looks good to me!

**Comment by thedavidprice** [comment]
So I'm having the most 🤬 time testing this one locally via either PR package install or RWT. Made some progress, but now I need to figure out why this is:
- duplicating Listr task title
- failing on create migration (I successfully ran the identical command after failure)

Screen Shot 2021-02-15 at 6 05 58 PM

**Comment by thedavidprice** [comment]
continues here https://github.com/redwoodjs/redwood/pull/1787

**Pull request merged**
